vue/.env.example

```
@@ -0,0 +1,4 @@
+# LaravelバックエンドのAPIベースURL
+# 開発環境の例: http://localhost:8002/api
+# 本番環境の例: https://api.example.com
+VITE_API=http://localhost:8002/api
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)